server/core/include/specColl.hpp

```diff
@@ -1,50 +1,77 @@
-/*** Copyright (c), The Regents of the University of California            ***
- *** For more information please refer to files in the COPYRIGHT directory ***/
+#ifndef SPEC_COLL_HPP
+#define SPEC_COLL_HPP
 
-/* specColl.h - header file for specColl.c
- */
+// This is needed for the SpecialCollPerm and because it is in lib,
+// it cannot be pulled into this file, unfortunately.
+#include "objInfo.h"
 
+struct DataObjInfo;
+struct DataObjInp;
+struct GenQueryOut;
+struct rodsObjStat;
+struct RsComm;
+struct SpecColl;
+struct SpecCollCache;
 
+extern "C"
+{
```

Comment:
This is not necessary since this is a C++ header file.

Reply:
Done